README.md

```diff
@@ -96,6 +92,78 @@ you'll need to add the following parameter to the `docker run` command above:
 Kubernetes on Azure AKS also requires a similar configuration at this time. Please see the
 `k8s-sample.yaml` file and [Kubernetes section](#kubernetes-troubleshooting) below for details.
 
+
+### Running in Podman
+
+Private Location Monitoring can also run in Podman, a Docker alternative.
+
+#### Prerequisites
+1. Podman v4.x.x+
+2. Linux Ubuntu 20.04+
+
+Installing Podman on Linux Ubuntu 20.04+
+
+Update your system and install Podman:
+
+```bash
+sudo apt-get update
+sudo apt-get -y install podman
+```
+
+Log in to Docker.io to pull the image:
+
+```bash
+podman login docker.io
+```
+
+You will be prompted to enter your Docker credentials.
+
+Pull the latest Uptime.com private location image:
+
+```bash
+podman pull docker.io/uptimecom/uptime-private-location:latest
```

Comment:
I think docker.io can be removed since all other places dont have it
podman pull /uptimecom/uptime-private-location:latest

Reply:
Podman doesn't pull from Docker Hub automatically, it needs to prefixed with `docker.io`